netmiko/utilities.py

```diff
@@ -183,7 +183,7 @@ def check_serial_port(name):
     """returns valid COM Port."""
     try:
         cdc = next(serial.tools.list_ports.grep(name))
-        return cdc.split()[0]
+        return cdc[0]
```

Comment:
This change I don't understand as it implies either: 1)there was a bug before and it didn't work, 2)pyserial changed, 3)something else is going on.

What is the value of `cdc` here?

Reply:
I think someone else reported this to me at one point, but I didn't have a good way to test it (I am not sure if it was you or someone else).

Reply:
@mreams13

Reply:
You opened the bug and suggested the fix. I think someone else might have suggested the bug on Slack. I am currently using netmiko over serial and I ran into the bug. I noticed it was still open with a fix included, but no one had actually changed it: 

"""
print(cdc) = COM6 - Prolific USB-to-Serial Comm Port (COM6)
print(cdc[0]) = COM6
print(cdc[1]) = Prolific USB-to-Serial Comm Port (COM6)
print(cdc[2]) = USB VID:PID=067B:2303 SER=5 LOCATION=1-6

ktbyers [8:14 PM]
Can you `print(type(cdc))`
"""